src/app-layout/__integ__/app-layout-toolbar-tooltips.test.ts

```diff
@@ -121,6 +120,42 @@ describe.each(['visual-refresh', 'classic'] as const)('%s', theme => {
         await expect(page.isExisting(`.${tooltipStyles.root}`)).resolves.toBe(false);
       })
     );
+
+    describe.each(['bottom', 'side'] as const)('splitPanelPosition=%s', splitPanelPosition => {
+      test(
+        'No split panel trigger tooltip showing for mouse interactions',
+        setupTest({ size, theme, splitPanelPosition }, async page => {
+          await expect(page.getElementsCount(`.${tooltipStyles.root}`)).resolves.toBe(0);
```

Comment:
Let's create a selector for tooltip root element

Reply:
Maybe we need to expose it as a part of our test utils? Would it be useful for consumers to have it as well?

Reply:
tooltipStyles comes is from an internal component and does not have its own selector to be used in a customer test-util. The internal component will need to be updated accordingly, this will best live in its own PR